src/events/txm_ad_events.js

```diff
@@ -12,6 +12,7 @@ const adEvents = {
     skipCardShown: 'skipCardShown',
     userCancel: 'userCancel',
     videoEvent: 'videoEvent',
+    xtendedViewStarted: 'xtendedViewStarted',
```

Comment:
this change means the TruexAdRenderer-HTML5 repo also needs to have the latest trues-shared
does this affect android-tv also, i.e. on the native side?

Reply:
Good point, I will add Android TV.